.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
```

Comment:
????

Reply:
Btw, I don't think, that IDE-specific files should be part of the repo.
We can create [.editorconfig](http://editorconfig.org/) instead, which is supported by all IDEs

Reply:
Maybe you are right, I just used to shared settings via .vscode in my other project. But here it seems to be useless, I have setup everything I need in global settings. 
We have no .editorconfig?? O_O oh god

Reply:
Si i'll delete both .vs and .vscode and add them to .gitignore ?

Reply:
@Le-Lutin yes, thank you.  :)